usaon_vta_survey/forms.py

```diff
@@ -83,6 +83,7 @@ def conv_String(self, field_args, **extra):
     ResponseObservingSystem: model_form(
         ResponseObservingSystem,
         only=common_response_object_fields,
+        field_args={'status': {'get_label': 'id'}},
```

Comment:
Should status be on the `Survey` form, not the `ResponseObservingSystem` form?

Reply:
@MattF-NSIDC  copy pasta mistake!

Reply:
removed this.